examples/pytorch/image-text/run_fuyu_no_trainer.py

```diff
@@ -0,0 +1,778 @@
+            args.model_name_or_path,
+            trust_remote_code=args.trust_remote_code,
+        )
+    else:
+        config = CONFIG_MAPPING[args.model_type]()
+        logger.warning("You are instantiating a new config instance from scratch.")
+
+    if args.tokenizer_name:
+        tokenizer = AutoTokenizer.from_pretrained(
+            args.tokenizer_name,
+            use_fast=not args.use_slow_tokenizer,
+            trust_remote_code=args.trust_remote_code,
+        )
+    elif args.model_name_or_path:
+        tokenizer = AutoTokenizer.from_pretrained(
+            args.model_name_or_path,
+            use_fast=not args.use_slow_tokenizer,
+            trust_remote_code=args.trust_remote_code,
+        )
+    else:
+        raise ValueError(
+            "You are instantiating a new tokenizer from scratch. This is not supported by this script. "
+            "You can do it from another script, save it, and load it from here, using --tokenizer_name."
+        )
+    tokenizer.pad_token = tokenizer.eos_token
+
+    # Load image_processor, in this script we only use this to get the mean and std for normalization.
+    processor = FuyuProcessor.from_pretrained(args.model_name_or_path)
+    tokenizer = processor.tokenizer
+    tokenizer.pad_token = tokenizer.eos_token
+    if args.model_name_or_path:
+        model = AutoModelForCausalLM.from_pretrained(
+            args.model_name_or_path,
+            from_tf=bool(".ckpt" in args.model_name_or_path),
+            config=config,
+            low_cpu_mem_usage=args.low_cpu_mem_usage,
+            trust_remote_code=args.trust_remote_code,
+        )
+    else:
+        logger.info("Training new model from scratch")
+        model = AutoModelForCausalLM.from_config(config, trust_remote_code=args.trust_remote_code)
+    # We resize the embeddings only when necessary to avoid index errors. If you are creating a model from scratch
+    # on a small vocab and want a smaller embedding size, remove this test.
+    embedding_size = model.get_input_embeddings().weight.shape[0]
+    if len(tokenizer) > embedding_size:
+        model.resize_token_embeddings(len(tokenizer))
+
+    # Preprocessing the datasets.
+    text_column_name = args.dataset_caption_column_name
+    image_column_name = args.dataset_image_column_name
+
+    def filter_corrupt_images(examples):
+        """remove problematic images"""
+        valid_images = []
+        for i in range(0, len(examples[image_column_name])):
+            try:
+                processor(text="test", images=[examples[image_column_name][i]])
+                valid_images.append(True)
+            except Exception:
+                valid_images.append(False)
+        return valid_images
+
+    with accelerator.main_process_first():
+        raw_datasets = raw_datasets.filter(
+            filter_corrupt_images,
+            batched=True,
+        )
+        # resize images
+        resize = Resize((224, 224))
```

Comment:
Why is this the only transformation applied? I'd expect some augmentation in a training script. You can look at the [image classification script as an example](https://github.com/huggingface/transformers/blob/39c3c0a72af6fbda5614dde02ff236069bb79827/examples/pytorch/image-classification/run_image_classification_no_trainer.py#L364).